include/operators/operator_stub.h

```diff
@@ -0,0 +1,15 @@
+#ifndef OPERATOR_STUB_H
```

Comment:
Just realised. Is this some legacy leftover?

Reply:
@nopeslide

Reply:
It is currently returned by the type resolver if no matching operator is found

Reply:
I expected the stub to have the same inputs as the operators, like the generic `operator_context`.

```c
struct operator_context
{
    operator_context_input      *input;
    operator_context_output     *output;
    operator_context_attribute  *attribute;
    operator_executer operator;
};
```

Reply:
thanks! didn't notice. fixed it in #15